drf_spectacular/contrib/django_filters.py

```diff
@@ -121,6 +121,12 @@ def resolve_filter_field(self, auto_schema, model, filterset_class, field_name,
         elif filter_field.label is not None:
             description = filter_field.label
 
+        if 'choices' in filter_field.extra:
+            description = description or ''
+            description += '>\n'
```

Comment:
- the `>` looks weird.
- we should add an empty line as separation if there is an description
- i don't like the way the empty backticks look

Reply:
I thought the '>' and backticks where part of the official format needed, but it looks like they can be removed/changed, committed that. I did add the '\n' but it doesn't seem to do anything for the json spec, not even sure if json supports multiline strings". 

My first attempt was by modifying the `build_choice_field` in `plumbing.py`, but these fields from DjangoFilters don't seem to pass through there. Or do you suggest to add code there as well as keeping it in the django filter part?